.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
+  "env": {
+    "commonjs": true,
+    "es6": true,
+    "node": true,
+    "mocha": true
```

Comment:
it’d be ideal to only enable this for test files, not everything else.

Reply:
ok